packages/bvaughn-architecture-demo/components/Icon.tsx

```diff
@@ -62,7 +62,7 @@ export default function Icon({
+        "M7.4342 23C7.64309 23 7.83706 22.951 8.01611 22.8531C8.20113 22.7552 8.41898 22.5931 8.66965 22.3667L12.0537 19.2735L17.6132 19.2827C18.5264 19.2827 19.3113 19.1022 19.9678 18.7412C20.6243 18.374 21.1256 17.8539 21.4718 17.1808C21.8239 16.5077 22 15.7061 22 14.776V7.50665C22 6.57656 21.8239 5.77497 21.4718 5.10188C21.1256 4.42879 20.6243 3.91173 19.9678 3.55071C19.3113 3.18357 18.5264 3 17.6132 3H6.38675C5.47359 3 4.68875 3.18357 4.03223 3.55071C3.38168 3.91173 2.88033 4.42879 2.5282 5.10188C2.17607 5.77497 2 6.57656 2 7.50665V14.776C2 15.7061 2.17905 16.5077 2.53715 17.1808C2.90122 17.8539 3.39958 18.371 4.03223 18.732C4.67084 19.093 5.40495 19.2735 6.23456 19.2735H6.48523V21.9169C6.48523 22.2474 6.56879 22.5105 6.7359 22.7063C6.90898 22.9021 7.14175 23 7.4342 23ZM7.6222 11.1413C7.6222 10.8232 7.72068 10.5631 7.91764 10.3612C8.12056 10.1531 8.37123 10.0491 8.66965 10.0491H10.9615V7.6994C10.9615 7.39345 11.06 7.13951 11.2569 6.93759C11.4539 6.72954 11.7046 6.62552 12.009 6.62552C12.3133 6.62552 12.564 6.72954 12.761 6.93759C12.9639 7.13951 13.0654 7.39345 13.0654 7.6994V10.0491H15.3662C15.6646 10.0491 15.9123 10.1531 16.1092 10.3612C16.3062 10.5631 16.4047 10.8232 16.4047 11.1413C16.4047 11.4473 16.3062 11.7043 16.1092 11.9123C15.9123 12.1143 15.6646 12.2152 15.3662 12.2152H13.0654V14.5649C13.0654 14.8709 12.9639 15.1248 12.761 15.3268C12.564 15.5287 12.3133 15.6296 12.009 15.6296C11.7046 15.6296 11.4539 15.5287 11.2569 15.3268C11.06 15.1248 10.9615 14.8709 10.9615 14.5649V12.2152H8.66965C8.37123 12.2152 8.12056 12.1143 7.91764 11.9123C7.72068 11.7043 7.6222 11.4473 7.6222 11.1413Z";
```

Comment:
Thanks! I think this is a better fit, although there's still a bit of vertical misalignment:
![Screen Shot 2022-08-10 at 1 39 02 PM](https://user-images.githubusercontent.com/29597/183979851-0a6828c7-7605-48bb-859b-0dbb2d867a0c.png)

Mostly mentioning as an FYI.

Reply:
Ah geeze lol fixed! 😊

Reply:
Thanks 😆